src/main/java/pl/marcin/project/AppRunnerConfig.java

```diff
@@ -0,0 +1,27 @@
+package pl.marcin.project;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
Do we need this annotation? 
Also, `Configuration` is usually used in classes that contain `@Beans` definitions. This one doesn't, actually this one has nothing to do with Spring :) I think we're safe to delete this annotation

Reply:
done